newIDE/app/src/SceneEditor/index.js

```diff
@@ -983,14 +983,14 @@ export default class SceneEditor extends React.Component<Props, State> {
   zoomIn = () => {
     const { editor } = this;
     if (editor) {
-      editor.zoomBy(0.1 * editor.getZoomFactor());
+      editor.zoomBy(1.1);
     }
   };
 
   zoomOut = () => {
     const { editor } = this;
     if (editor) {
-      editor.zoomBy(-0.1 * editor.getZoomFactor());
+      editor.zoomBy(0.9);
```

Comment:
`0.9 * 1.1 = 0.99` which is not `1`.
May I suggest `2**(1/8) = 1.09050773266526` and `2**(-1/8) = 0.917004043204671`?

Reply:
Good remark. I'll see what I can do